docs/en_US/plugins/plugins_tutorial.md

```diff
@@ -26,46 +26,31 @@ These limitations are relatively strict, and they almost require compiling and r
 
-### Create and develop the plugin project
-
-There are some plugin examples in the *extensions* directory of the eKuiper project source code. The user customized plugin can also be developed in the eKuiper project. However, users usually need to create the new project outside of the eKuiper project to develop customized plugins, to manage code more conveniently. It's recommended to use Go module to develop plugin projects, the typical structure of project is listed as following.
-
-```
-plugin_project
-  sources         //source code directory of the plugin source 
-    mysource.go
-  sinks           //source code directory of the plugin sink
-    mysink.go
-  functions       //source code directory of the plugin function
-    myfunction.go
-  target          //directory of compiling results   
-  go.mod          //file go module
-```
-
-Developing a plugin needs to extend the interface in eKuiper, so it must depend on the eKuiper project. The simplest go.mod also needs to include the dependency for eKuiper. A typical go.mod is as follows:
-```go
-module samplePlugin
-
-go 1.16
-
-require (
-	github.com/lf-edge/ekuiper v0.0.0-20200323140757-60d00241372b
-)
-```
-
-The eKuiper plugin has three types. The source code can be put into the corresponding directory. For the detailed method of plugin development: [LF Edge eKuiper extension](../extension/overview.md). This article will take the Sink plugin as an example to introduce the process of plugin development and deployment. We will develop a basic MySql sink, for write stream output data to the MySql database.
-
-- Create plugin the project samplePlugin with the above directory structure
+- Create the plugin project
 - Create file mysql.go under the sinks directory
 - Edit file mysql.go for implementing the plugin
-    -  Implement [api.Sink](https://github.com/lf-edge/ekuiper/blob/master/pkg/api/stream.go) interface
+    - Implement [api.Sink](https://github.com/lf-edge/ekuiper/blob/master/pkg/api/stream.go) interface
     - Export Symbol: Mysql. It could be a constructor function so that each rule can instantiate an own mysql plugin instance. Or it could be the struct which means every rule will share a singleton of the plugin. If the plugin has states like the connection, the first approach is preferred.
 - Edit go.mod, add Mysql driver module
+- Build the eKuiper and plugin
 
-The complete source code of mysql.go is as follows:
+### Create plugin project inside eKuiper
 
+When user develop the plugin by this way, he must firstly download the eKuiper source code and run `make` command inside the eKuiper root directory. There are some plugin examples in the *extensions* directory of the eKuiper project source code. The advantage is all existing official plugins are developed by this way so new plugin developer can directly start instead of setting up a new project from scratch.
```

Comment:
When users develop or when the user develops.  This problem also happens elsewhere, search user to check